crypto/rand/rand_unix.c

```diff
@@ -127,27 +135,35 @@ int RAND_poll_ex(RAND_poll_cb rand_add, void *arg)
 #  endif
 
 /*
- * Try the various seeding methods in turn, exit when succesful.
+ * Try the various seeding methods in turn, exit when successful.
+ *
+ * TODO(DRBG): If more than one entropy source is available, is it
+ * preferrable to stop as soon as enough entropy has been collected
+ * (as favored by @rsalz) or should one rather be defensive and add
+ * more entropy than requested and/or from different sources?
```

Comment:
Perhaps mention the case when sufficient entropy has already been collected to meet the request and a later source is blocking.

The user has indicated trust in the sources (by enabling them) which is also a factor.

Reply:
I extended the comment a little, but in general I consider that comment only a kick starter for further discussions. To be honest, I was hoping that you or @kroeckx would come up with some suggestions. Because I don't have a good proposal at the moment, except for the following: Add an API to register your own `acquire_entropy((RAND_POOL * pool)` callbacks and provide

1. a default implementation of this callback, and
2. a toolbox of standard implementations like `RAND_POOL_acquire_entropy_getrandom()` which the user can call from his callback.

We should discuss this elsewhere, I would like to get this PR off the shelf as soon as possible.

Reply:
I've a couple of ideas but nothing concrete:  I agree we should discuss this elsewhere: #4394